changelog.d/13831.misc

```diff
@@ -0,0 +1 @@
+Add experimental support for tracking device IDs of pushers (as a partial implementation of [MSC3881](https://github.com/matrix-org/matrix-spec-proposals/pull/3881)).
```

Comment:
We might want to massage the changelog a bit to have something unified with https://github.com/matrix-org/synapse/pull/13799 if we ship both in the same version of Synapse (depending when reviews happen).